### DISCUSSION TITLE
[Feat]: Ability to control scroll only with API while disabling completely the slider to the user

### DISCUSSION THREAD

**Discussion by jagonzalr** [davidjerleke/embla-carousel#918] [Help] [answered]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

You can set a carousel active/inactive by passing the option `active` when initializing it. This is great if you don't want the users to be able to interact with it. What it blocks is also the API, so as a dev I want to be able to dynamically go to a different slide while the user can't do anything. From what I can see in this [line](https://github.com/davidjerleke/embla-carousel/blob/master/packages/embla-carousel/src/components/EmblaCarousel.ts#L159) it checks if the carousel is active or not before trying to scroll. Could we add a secondary option, eg. `activeApi` where it could check if the API can interact with the carousel even if it is inactive for the user?

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment]
@jagonzalr thanks for your question. Do you want to disable drag events? 

The `active` option is probably not what you're looking for. It's useful if you want to temporarily disable the carousel and the main API. For example, when you want to disable/activate it on certain breakpoints.

**Reply by jagonzalr** [reply]
Yeah I want to disable drag events, couldn't find anything on the docs on how to do that 🤔

**Reply by davidjerleke** [reply, marked as answer]
@jagonzalr use this option: `watchDrag: false`.

**Reply by jagonzalr** [reply]
Working like a charm @davidjerleke. Thanks for being so fast at helping me :)

**Reply by davidjerleke** [reply]
Docs reference: https://www.embla-carousel.com/api/options/#watchdrag
